src/axiom/universal_interpreter.py

```diff
@@ -20,15 +32,16 @@
 1.  **ONLY use the information given in the 'Facts' string.**
 2.  **DO NOT add any extra information, commentary, or meta-analysis.**
 3.  **DO NOT apologize or mention your own limitations.**
-4.  **Your output must be ONLY the rephrased sentence and nothing else.**"""[1:]
+4.  **Your output must be ONLY the rephrased sentence and nothing else.**
+"""[1:]
```

Comment:
If you are intending prompt to not have trailing newline, need to do `[1:-1]` instead now.